centreon/packages/ui/src/Colors/index.tsx

```diff
@@ -0,0 +1,165 @@
+import { makeStyles } from 'tss-react/mui';
+import { includes, toPairs } from 'ramda';
+
+import { Stack, Box, useTheme, Typography, Divider, Grid } from '@mui/material';
+
+import { ThemeMode } from '@centreon/ui-context';
+
+const useStyles = makeStyles()((theme) => ({
+  divider: {
+    margin: theme.spacing(2)
+  },
+  headerContainer: {
+    display: 'flex',
+    justifyContent: 'space-between'
+  },
+  itemContainer: {
+    borderRadius: 1,
+    margin: theme.spacing(2),
+    padding: theme.spacing(1),
+    textAlign: 'center'
+  },
+  itemContainerText: {
+    border: '0.3px solid',
```

Comment:
I think you can remove the border in order to avoid visual confusion
```suggestion
    border: 'none',
```